### PR TITLE
Task/fix elements

### PR DESCRIPTION
**Pull request by paulmskim**

### CODE REVIEW

File: plugins/events-pro/src/modules/elements/series-ends/element.js [hunk 1]
```diff
@@ -45,12 +52,28 @@ const onSeriesEndsChange = ( ownProps, edit ) => ( selectedOption ) => (
...
 
-const onSeriesEndsOnDateChange = ( ownProps, edit ) => ( date ) => {
-	const endDate = date
-		? momentUtil.toDate( momentUtil.toMoment( date ), dateUtil.FORMATS.DATABASE.datetime )
-		: '';
-	edit( ownProps.index, { [ KEY_END_DATE ]: endDate } );
-};
+const onSeriesEndsOnDateChange = ( ownProps, edit, end ) => (
+	( date, modifiers, dayPickerInput ) => {
+		console.log('here in day change');
```
Comment (nealfennimore) [inline plugins/events-pro/src/modules/elements/series-ends/element.js]: :video_game:

File: plugins/events-pro/src/modules/elements/series-ends/template.js [hunk 2]
```diff
@@ -35,6 +35,8 @@ const SeriesEnds = ( {
 
 		if ( seriesEnds && seriesEnds.value === constants.DATE ) {
 			const seriesEndsOnDateObj = new Date( seriesEndsOnDate );
+			console.log( 'seriesEndsOnDate', seriesEndsOnDate );
```
Comment (nealfennimore) [inline plugins/events-pro/src/modules/elements/series-ends/template.js]: :video_game:

File: plugins/events-pro/src/modules/data/blocks/recurring/selectors.js [hunk 3]
```diff
@@ -25,27 +48,37 @@ export const getRule = createSelector(
 
 export const getType = createSelector(
 	[ getRule ],
-	( rule ) => rule.type,
+	( rule ) => rule[ KEY_TYPE ],
```
Comment (nealfennimore) [inline plugins/events-pro/src/modules/data/blocks/recurring/selectors.js]: I'll make a note to do this with exception selectors. Probably will move the selectors to shared folder and then import into here for easier reuse

Comment (paulmskim) [reply plugins/events-pro/src/modules/data/blocks/recurring/selectors.js]: yeah like 95% of these selectors are the same, so would be nice to get them using the same, just pass in an exception selector or recurrence selector.

**Pull request merged**